DMX.Core.Api/Brokers/LabApis/ExternalLabApiBroker.cs

```diff
@@ -42,12 +44,20 @@ private IRESTFulApiFactoryClient GetApiClient(IConfiguration configuration)
             return new RESTFulApiFactoryClient(this.httpClient);
         }
 
-        private static string GetApiAccessToken(IConfiguration configuration)
+        private static string GetAvailableDevicesAccessToken(IConfiguration configuration)
         {
             LocalConfigurations localConfigurations =
                 configuration.Get<LocalConfigurations>();
 
-            return localConfigurations.ApiConfigurations.AccessKey;
+            return localConfigurations.ApiConfigurations.GetAvailableDevicesAccessKey;
+        }
+
+        private static string GetAllDevicesAccessToken(IConfiguration configuration)
+        {
+            LocalConfigurations localConfigurations =
+                configuration.Get<LocalConfigurations>();
+
+            return localConfigurations.ApiConfigurations.GetAllDevicesAccessKey;
```

Comment:
Rename this. this is a property not an action/method. It should just say `AllDevicesAccessKey`